fluent.runtime/README.rst

```diff
@@ -0,0 +1,19 @@
+fluent.runtime |fluent.runtime|
+===============================
+
+Use `Fluent`_ to localize your Python application. It comes with a ``Localization``
+class to use, based on an implementation of bundle. It uses the parser from
```

Comment:
The way it's used here, `bundle` is a bit enigmatic. Perhaps mention `FluentBundle`, i.e. the class?

(I realize this is copied from the current README; if you'd prefer to fix this later rather than in a drive-by, that's fine with me.)

Reply:
Fixed this in both READMEs.